core/admin/mailu/models.py

```diff
@@ -35,11 +35,14 @@ class IdnaEmail(db.TypeDecorator):
     impl = db.String(255, collation="NOCASE")
 
     def process_bind_param(self, value, dialect):
-        localpart, domain_name = value.split('@')
-        return "{0}@{1}".format(
-            localpart,
-            idna.encode(domain_name).decode('ascii'),
-        )
+        try:
+            localpart, domain_name = value.split('@')
+            return "{0}@{1}".format(
+                localpart,
+                idna.encode(domain_name).decode('ascii'),
+            )
+        except:
```

Comment:
I think if we catch all exceptions it is maybe a little bit overkilling? IMHO we should just catch the specific exception.

```suggestion
        except ValueError:
```